### ISSUE TITLE
Late game items too close to End game

### ISSUE THREAD

**Issue by gamerpaddy** [StrangeLoopGames/EcoSuggestions#739]
It tages forever to get to the steam cart, it takes almost as long to get the chainsaw and excavator. 
but between them and the final end lasers its just a matter of minutes.  

it would be nice to make the endgame a bit apart from the late game. 
many people stop playing when the asteroid came down. this way they would keep playing a lot longer.

**Comment by Blanclour** [comment]
I agree, building the lasers to destroy the asteroid should be far more expensive and possibly require a separate progression tree of crafting tables that comes after the industry crafting tables.

**Comment by johnkslg** [comment]
agreed!
